### DISCUSSION TITLE
How to reset product data or delete product data in bulk.

### DISCUSSION THREAD

**Discussion by snmin** [YetiForceCompany/YetiForceCRM#15763] [How to]
Hello, my product field settings are wrong and a lot of product data has been imported, how can I reset the product data.
Or how can I delete product data in batches? Instead of only selecting 20 product data at a time to delete. Thanks! !

**Comment by rskrzypczak** [comment]
Create a filter and select all as shown below.
![image](https://user-images.githubusercontent.com/10221999/156302822-f1448e57-a13d-46d1-8d1a-cfe69a54e486.png)
more: https://github.com/YetiForceCompany/YetiForceCRM/issues/15765
